docs/operations/data_migration.md

```diff
@@ -838,3 +838,45 @@ Example: `cf ssh getgov-za`
 
 ### Running locally
 ```docker-compose exec app ./manage.py populate_domain_request_dates```
+
+## Create federal portfolio
+This script takes the name of a `FederalAgency` (like 'AMTRAK') and does the following:
```

Comment:
The additional context of when this may fail etc should probably be in here as well (in a summed up format).